pkg/generate/app/cmd/newapp_test.go

```diff
@@ -76,9 +76,9 @@ func TestAddArguments(t *testing.T) {
 			unknown: []string{},
 		},
 		"mix 1": {
-			args:       []string{"git://server/repo.git", "mysql+ruby~git@test.server/repo.git", "env1=test", "ruby-helloworld-sample"},
-			repos:      util.StringList([]string{"git://server/repo.git"}),
-			components: util.StringList([]string{"mysql+ruby~git@test.server/repo.git", "ruby-helloworld-sample"}),
+			args:       []string{"git://github.com/openshift/origin.git", "mysql+ruby~git@github.com/openshift/origin.git", "env1=test", "ruby-helloworld-sample"},
+			repos:      util.StringList([]string{"git://github.com/openshift/origin.git"}),
+			components: util.StringList([]string{"mysql+ruby~git@github.com/openshift/origin.git", "ruby-helloworld-sample"}),
```

Comment:
actually, it is not a `git clone` but a `git ls-remote` ... it is lightweight/super fast